data/tlg0007/tlg077/tlg0007.tlg077.perseus-eng4.xml

```diff
@@ -0,0 +1,1140 @@
+      		<publisher>Trustees of Tufts University</publisher>
+      		<pubPlace>Medford, MA</pubPlace>
+      		<authority>Perseus Project</authority>
+      		<date type="release">2010-12-13</date>
+      	</publicationStmt>
+         <sourceDesc>
+            <biblStruct>
+               <monogr>
+                  <author>Plutarch</author>
+                  <title>Plutarch's Morals.</title>
+                  <respStmt>
+                     <resp>Translated from the Greek by several hands. Corrected and revised	by</resp>
+                     <name>William W. Goodwin, PH. D.</name>
+                  </respStmt>
+                  <imprint>
+                     <pubPlace>Boston</pubPlace>
+                     <publisher>Little, Brown, and Company</publisher>
+                     <pubPlace>Cambridge</pubPlace>
+                     <publisher>Press Of John Wilson and son</publisher>
+                     <date>1874</date>
+                  </imprint>
+                  <biblScope unit="volume">1</biblScope>
+               </monogr>
+            	<ref target="https://archive.org/details/plutarchsmoralst01plutuoft/page/250">The Internet Archive</ref>
+            </biblStruct>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+      	<p>Text encoded in accordance with the latest EpiDoc standards</p>
+      	<p>The following text is encoded in accordance with EpiDoc standards and with the CTS/CITE Architecture</p>
+         <refsDecl n="CTS">
+         <cRefPattern n="section" matchPattern="(\w+)" replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1'])"> 
+         <p>This pointer pattern extracts sections</p></cRefPattern>
+            <!--<state unit="section"/>-->
+         </refsDecl><!--<refsDecl doctype="TEI.2">
+				<state unit="stephpage" n="chunk"/>
+				</refsDecl>--></encodingDesc>
+      <profileDesc>
+         <langUsage>
+            <language ident="eng">English</language>
+            <language ident="grc">Greek</language>
+         	<language ident="lat">Latin</language>
+         </langUsage>
+      	<particDesc>
+      		<person><persName>Moschion</persName></person>
+      		<person><persName>Zeuxippus</persName></person>
+      	</particDesc>
+      </profileDesc>
+      <revisionDesc>
+      	<change who="GRC" when="2006">tagging</change>
+      	<change who="Karina Cooper" when="2019-06-27">EpiDoc and CTS conversion</change>
+      </revisionDesc>
+   </teiHeader>
+   <text xml:lang="eng">
+      <body>
+      	<div type="translation" xml:lang="eng" n="urn:cts:greekLit:tlg0007.tlg077.perseus-eng4">
+         <pb xml:id="v.1.p.251"/>
+      		<head>Plutarch's Rules for the Preservation of Health.</head><lb/>
+      		<head>A Dialogue.</head><lb/>
+      		<head>MOSCHIO, ZEUXIPPUS.</head>
+         <div type="textpart" subtype="section" n="1">
+         	<p rend="indent"><said who="#Moschion"><label>MOSCHION.</label> And you, Zeuxippus, diverted Glaucus the
```

Comment:
I think that this translation calls him Moschio? So the tag and label should reflect that.